src/resonanceReconstruction/breitWigner/multiLevel/Apply/src/call.hpp

```diff
@@ -2,11 +2,85 @@ template< typename Functor >
 decltype(auto)
 operator()( const ENDF::ResonanceRange& range,
             Functor&& callback ) const {
+
   try {
+    const EnergyRange energyRange {
+      range.EL() * electronVolts,
+      range.EH() * electronVolts
+    };
+    const auto mlbw = std::get< ENDF::resolved::MLBW >( range.parameters() );
+
+    /*
```

Comment:
So, we don't want to have lengthy comments like this in our code. Having the description of why we changed things should go in a Pull Request description or conversation. It doesn't make sense to have it permanently in the code because "tomorrow" it won't be correct or meaningful.

Reply:
OK, I should talk with someone about how the new build works. I'd like to do this sooner rather than later, as I have other work to do, and don't want this to sit for too long. I also have some ideas on what the slow-compile issue might be, and really want to check it out.

Does your comment on "this->" mean y'all would prefer I just left it in?

As usual, git is feasible to use if only one person is working with it, but when multiple people are involved, it becomes so awkward to deal with conflicts that it's unusable. It looks like I'll be saving my work locally, checking out a brand-new version, cutting-and-pasting what I've done, and basically redoing the whole thing.

If I check out the brand new, latest master, absolutely fresh, what sequence of commands is needed to build and test? Or is that documented somewhere?

Reply:
The commands to build a component is the same with the new build system as with the old system. The new system is just (much) better about handling dependencies. In essence, metaconfigure no longer exists.

Reply:
You can do (when in your branch) `git merge master`. You will have to resolve some conflicts, but that's regular workflow when using git.

Reply:
So, then, I just completely leave out the "metaconfigure/generate.py cmake" command?

Reply:
Yep, no longer needed. If you add a new test file, you have to modify the CMakeLists.txt file manually. @nathangibson14  is working on scripts to automate that too.

Reply:
I know about the git merge. In cases like this, so many conflicts are reported, and git clutters the code so much with markup, that it's easier and safer to just toss everything and cut-and-paste my changes from elsewhere.